produce_snp_matrix.py

```diff
@@ -1,46 +1,86 @@
 from glob import glob
-import subprocess
+import json
 import argparse
 import utils
 import os
 
-def snps(input_path, output_path):
+def download_test_case(case, download_path):
+    with open('./test.json') as f:
+        x =json.load(f)
+
+        try:
```

Comment:
I think everything from here is indented to the same level as line 9. That means it's all within the context of the open statement on line 8. I think it should actually be indented to the same level as line 8. Thus the file will be closed after `x' is assigned on line 9.